server/submissionWorkflow/model.ts

```diff
@@ -0,0 +1,27 @@
+export default (sequelize, dataTypes) => {
+	return sequelize.define(
+		'SubmissionWorkflow',
+		{
+			id: sequelize.idType,
+			enabled: { type: dataTypes.BOOLEAN },
+			instructions: { type: dataTypes.JSONB },
+			emailText: { type: dataTypes.JSONB },
+			layoutBlock: {
+				type: dataTypes.JSONB,
+			},
+			targetEmailAddress: { type: dataTypes.STRING },
+		},
+		{
+			classMethods: {
+				associate: (models) => {
+					const { Collection, SubmissionWorkflow } = models;
+					SubmissionWorkflow.belongsTo(Collection, {
```

Comment:
So I know these Sequelize association method names are actively misleading. One tip for using them is that, when using `belongsTo`, all of these things need to match up:

- The model provided as the first argument
- The name of the `as` property
- The foreign key

Here you have a mismatch, that results in a `submissionWorkflow.submissionWorkflow` property which is actually a Collection. If you change `as` to `'collection'` you'll get `submissionWorkflow.collection`, which is a useful thing to have. To make this work you'll also need to add a (nullable) `collectionId` property to this model, which I wanna say you previously had?

Also, let's drop the `onDelete: 'CASCADE'`. If a Collection is deleted, I think we want to err on the side of keeping its submission workflow around in the database.